languages/elixir/exercises/concept/booleans/mix.exs

```diff
@@ -5,7 +5,7 @@ defmodule Rules.MixProject do
     [
       app: :booleans,
       version: "0.1.0",
-      elixir: "~> 1.10",
+      # elixir: "~> 1.10",
```

Comment:
I went ahead and committed this to [be bold](https://en.wikipedia.org/wiki/Wikipedia:Be_bold), but open to further discussion on if this is the right call.

Reply:
I think in a mentorless enviroment, we should consider pushing the latest minor release.

Reply:
Since we will be relying on the test runner and compiler more to catch thjngs

Reply:
Won't this require students preferring local dev and CLI to potentially update their elixir versions to the latest (or just comment out the line)? For the web, we could still run the latest release even with no elixir version enforced in this file.

Reply:
My thought though is, where is it communicated to students that we will run their solution on 1.10 (or whatever version) and it fails because an older version didn't raise an error appropriately?

If it's commented out, will that be enough to communicate a potential disparity between test runner and local results?

I'm sure many languages have this issue (rust comes to mind because it is in a fast revision cycle). Or even Java now that they are in a 6 month feature release cycle.  Maybe I could raise this as an issue to discuss in the next exercism v3 meetup?

Reply:
The beauty of backwards compatibility / non-breaking change is that if a student writes code in elixir 1.0, we can still compile and test that code with elixir 1.10. Elixir also pays extra attention to this, so I don't anticipate that we will have problems with students submitting code from older versions. If anything the opposite will more likely be true... if we don't stay on top of upgrading, students may use features introduced in minor versions that we have not yet upgraded to.

Reply:
I agree, it is nice that extra attention has been made. But I think I described the situation I had with a student running 1.9 found that the compiler wasn't checking @spec for undeclared custom struct types, where as on 1.10 there was a compiler error. 

I just don't want to run into a rare situation where students get passing code on their machine but fails on the test runner. I would believe that the conversion rate in those events for a student to reach out to catch this bug might be very low and go unnoticed.

Reply:
I think the general concern of addressing errors that occur in our builds is very much worth solving, but I don't think that locking the elixir minor version is the right way. Since patch versions are for bugfixes, 1.10.0, which would be a valid version under this constraint, may not have the bugfix (but instead 1.10.1 or 2). This could be an erlang bug, at least in principle, which isn't affected by this constraint. I like the more general approach of communicating the error back to the student and along with the versions our build uses, plus an escape hatch to talk to a human to help figure it out.

Reply:
my plan is to discuss this in the video-meetup tomorrow morning, will ask about the details about how versions are planned to be communicated and what kind of escape hatches are available to users in th event of unintelligible/unexpected output